### PR TITLE
Update bundle for T2 extensions

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "id": "Microsoft.Azure.WebJobs.Extensions.Storage",
-    "version": "5.0.0-beta.2",
+    "version": "5.0.0-beta.4",
     "name": "AzureStorageQueues",
```
Comment (JoshLove-msft) [inline src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json:5]: Is this name correct?

File: src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },
   {
     "id": "Microsoft.Azure.WebJobs.Extensions.Storage.Blobs",
-    "version": "5.0.0-beta.2",
+    "version": "5.0.0-beta.4",
     "name": "AzureStorageQueues",
```
Comment (JoshLove-msft) [inline src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json:16]: I think this got swapped with the blobs binding.

File: src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 [
   {
     "id": "Microsoft.Azure.WebJobs.Extensions.Storage",
-    "version": "5.0.0-beta.2",
-    "name": "AzureStorageQueues",
+    "version": "5.0.0-beta.4",
+    "name": "AzureStorage",
     "bindings": [
```
Comment (paulbatum) [inline src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json:6]: @soninaren I am unsure about the importance of the name element here.  Is there any behavior it impacts?

Comment (soninaren) [reply src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json:6]: No this is just a config file used to generate bundles. Name does not have behavioral impact.

**Comment by JoshLove-msft** [comment]
Note, I don't have write access for this repo, so I think one of you will need to merge it.

**Review by paulbatum — APPROVED** [review]

**Pull request merged**